a11y/uk/gov/hmrc/merchandiseinbaggage/FrontendAccessibilitySpec.scala

```diff
@@ -0,0 +1,105 @@
+    case cannotUseServiceIrelandView: CannotUseServiceIrelandView => render(cannotUseServiceIrelandView)
+    case cannotUseServiceView: CannotUseServiceView => render(cannotUseServiceView)
+    case checkYourAnswersAmendExportView: CheckYourAnswersAmendExportView => render(checkYourAnswersAmendExportView)
+    case checkYourAnswersAmendImportView: CheckYourAnswersAmendImportView => render(checkYourAnswersAmendImportView)
+    case checkYourAnswersExportView: CheckYourAnswersExportView => render(checkYourAnswersExportView)
+    case checkYourAnswersImportView: CheckYourAnswersImportView => render(checkYourAnswersImportView)
+    case customsAgentView: CustomsAgentView => render(customsAgentView)
+    case declarationConfirmationView: DeclarationConfirmationView => render(declarationConfirmationView)
+    case declarationNotFoundView: DeclarationNotFoundView => render(declarationNotFoundView)
+    case enterEmailView: EnterEmailView => render(enterEmailView)
+    case enterOptionalEmailView: EnterOptionalEmailView => render(enterOptionalEmailView)
+    case eoriNumberView: EoriNumberView => render(eoriNumberView)
+    case errorTemplate: ErrorTemplate                                           => render(errorTemplate)
+    case exciseAndRestrictedGoodsView: ExciseAndRestrictedGoodsView => render(exciseAndRestrictedGoodsView)
+    case goodsDestinationView: GoodsDestinationView => render(goodsDestinationView)
+    case goodsInVehicleView: GoodsInVehicleView => render(goodsInVehicleView)
+    case goodsOriginView: GoodsOriginView => render(goodsOriginView)
+    case goodsOverThresholdView: GoodsOverThresholdView => render(goodsOverThresholdView)
+    case goodsRemovedView: GoodsRemovedView => render(goodsRemovedView)
+    case goodsTypeView: GoodsTypeView => render(goodsTypeView)
+    case goodsVatRateView: GoodsVatRateView => render(goodsVatRateView)
+    case importExportChoice: ImportExportChoice => render(importExportChoice)
+    case journeyDetailsPage: JourneyDetailsPage => render(journeyDetailsPage)
+    case newOrExistingView: NewOrExistingView => render(newOrExistingView)
+    case noDeclarationNeededView: NoDeclarationNeededView => render(noDeclarationNeededView)
+    case paymentCalculationView: PaymentCalculationView => render(paymentCalculationView)
+    case previousDeclarationDetailsView: PreviousDeclarationDetailsView => render(previousDeclarationDetailsView)
+    case progressDeletedView: ProgressDeletedView => render(progressDeletedView)
+    case purchaseDetailsExportView: PurchaseDetailsExportView => render(purchaseDetailsExportView)
+    case purchaseDetailsImportView: PurchaseDetailsImportView => render(purchaseDetailsImportView)
+    case removeGoodsView: RemoveGoodsView => render(removeGoodsView)
+    case retrieveDeclarationView: RetrieveDeclarationView => render(retrieveDeclarationView)
+    case reviewGoodsView: ReviewGoodsView => render(reviewGoodsView)
```

Comment:
also here I see you have not add 3 views:
- case serviceTimeoutView: ServiceTimeoutView                           => render(serviceTimeoutView)
- case sessionExpiredView: SessionExpiredView                           => render(sessionExpiredView)
- case testOnlyDeclarationJourneyPage: TestOnlyDeclarationJourneyPage   => render(testOnlyDeclarationJourneyPage)

In order to make `TestOnlyDeclarationJourneyPage` working you need to add implicit for a form of string like
`private val stringForm: Form[String]             = Form("declarationJourney" -> nonEmptyText)`
`implicit val arbFormString: Arbitrary[Form[String]]                          = fixed(stringForm)`

There is nothing wrong to test these as they dont have errors now